Interface/Directory.Build.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+  <PropertyGroup>
+    <!-- Force malformed versions to be an error -->
+    <WarningsAsErrors>$(WarningsAsErrors);CS7035</WarningsAsErrors>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(ContinuousIntegrationBuild)' == 'true'">
+    <Version Condition="$(CiBuildVersion.StartsWith('api'))">$(CiBuildVersion.Substring(3))</Version>
```

Comment:
We are going to strongly recommend we use a single release version for device repos (prefixed with `v`, see https://github.com/harp-tech/protocol/issues/145).

We probably don't need to change this right now anyway, but just a heads up in case it is relevant.

Reply:
Yes, this PR (and the others like it) were intended only to unblock publishing packages for Bonsai 2.9. It's not meant to be even *slightly* forward-looking.